SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+We advise users to run the most recent release of the NGINX Kubernetes Gateway. A table with the supported versions can be found in the [README](README.md#technical-specifications).
+
+## Reporting a Vulnerability
+
+The F5 Security Incident Response Team (F5 SIRT) has an email alias that makes it easy to report potential security vulnerabilities.
+
+- If you’re an F5 customer with an active support contract, please contact [F5 Technical Support](https://www.f5.com/services/support).
```

Comment:
I don't think we should mention support contracts here since NKG does not currently offer a support package.

Reply:
I think the support contract is in general, not necessarily for this project? But I have no idea how support contract work, I just copied the standard message and I'm happy to remove it if you think it's misleading.

Reply:
I think we should remove it. In our README we state that NKG isn't covered https://github.com/nginxinc/nginx-kubernetes-gateway#support

Reply:
updated it